docs/cases/cases-ui-integrations.asciidoc

```diff
@@ -34,6 +34,43 @@ image::images/cases-ui-connector.png[Shows the page for creating connectors]
 - {kibana-ref}/swimlane-action-type.html[{swimlane} connector]
 - {kibana-ref}/cases-webhook-action-type.html[{webhook-cm} connector]
 
+[float]
+[[mapped-case-fields]]
+=== Mapped case fields
+
+To represent an {elastic-sec} case in an external system, case fields are mapped to existing fields in {sn}, {jira}, {ibm-r}, and {swimlane}. For the {webhook-cm} connector, case fields can mapped to custom or pre-existing fields in the external system you're connecting to.
```

Comment:
```suggestion
When you export an {elastic-sec} case to an external system, case fields are mapped to existing fields in {sn}, {jira}, {ibm-r}, and {swimlane}. For the {webhook-cm} connector, case fields can be mapped to custom or pre-existing fields in the external system you're connecting to.
```

Reply:
I like this, thanks @benironside ! The language is simpler and easier to mentally picture.